src/Api.php

```diff
@@ -133,6 +135,23 @@ private function baseApiUri()
                                            : static::LIVE_API_URI;
     }
 
+    /**
+     * Gets the request parameter type for the Guzzle client request.
+     *
+     * @param string $method
+     *
+     * @return string
+     */
+    private function requestParameterType(string $method): string
+    {
+        switch (strtolower($method)) {
+            case 'post':
+                return 'form_params';
+            default:
```

Comment:
Would probably put this in a `case` for `GET`, just so it’s not passed for other HTTP methods i.e. `PUT`, `DELETE` etc.